erpnext/accounts/party.py

```diff
@@ -303,7 +303,13 @@ def validate_party_accounts(doc):
 			company_default_currency = frappe.db.get_value('Company', account.company, "default_currency")
 
 		if existing_gle_currency and party_account_currency != existing_gle_currency:
-			frappe.throw(_("Accounting entries have already been made in currency {0} for company {1}. Please select a receivable or payable account with currency {0}.").format(existing_gle_currency, account.company))
+			if doc.doctype == 'Customer':
```

Comment:
@anupamvs There is a `validate_party_gle_currency` function just above this function which can be used here to validate party gle currency and avoid code duplication.

Reply:
@deepeshgarg007 Done!